File: app/src/main/java/com/drivingschoolrwandaapp/models/request/VerifyOtpRequest.java
package com.drivingschoolrwandaapp.models.request;

import com.google.gson.annotations.SerializedName;

public class VerifyOtpRequest {

    @SerializedName("email")
    private String email;

    @SerializedName("otp")
    private String otp;

    public VerifyOtpRequest(String email, String otp) {
        this.email = email;
        this.otp = otp;
    }

    public String getEmail() {
        return email;
    }

    public String getOtp() {
        return otp;
    }
}

File: app/src/main/java/com/drivingschoolrwandaapp/models/request/VerifyOtpRequest.kt
package com.drivingschoolrwandaapp.models.request

import com.google.gson.annotations.SerializedName

data class VerifyOtpRequest(
    @SerializedName("email") var email: String = "",
    @SerializedName("otp") var otp: String = ""
)
